src/Logger/Log.php

```diff
@@ -48,14 +48,14 @@ class Log
     protected string $action;
 
     /**
-     * @var array (optional)
+     * @var array<string, string> (optional)
      */
     protected array $tags = [];
 
     /**
      * @var array (optional)
      */
-    protected array $extra = [];
+    protected array $extra = []; /** @phpstan-ignore-line */
```

Comment:
Please add the proper type rather than ignoring error. Looks like the type should be:

```php
    /**
     * @var array<string, mixed> (optional)
     */
    protected array $extra = [];
```

Reply:
@stnguyen90 I am encountering few issues in the actual code when I set value as mixed type.

Reply:
What issues? They should probably be resolved.